calculator/src/App.js

```diff
@@ -0,0 +1,83 @@
+/* eslint no-eval: 0 */
+import React, { Component } from 'react';
+import './App.css';
+import Results from './components/Results';
+import Buttons from "./components/Buttons";
+
+class App extends Component {
+    constructor(){
+        super();
+
+        this.state = {
+            result: ""
+        }
+    }
+
+    resultFunction = () => {
+        let checkResult = ''
+        if(this.state.result.includes('--')){
+            checkResult = this.state.result.replace('--','+')
+        }
+
+        else {
+            checkResult = this.state.result
+        }
+
+        try {
+            this.setState({
+                result: (eval(checkResult) || "" ) + ""
```

Comment:
eval is a good function, but it is not the ideal solution for this app. Eval automatically rounds up which is not as accurate as a calculator should be